eox_tenant/backends/database.py

```diff
@@ -180,3 +180,191 @@ def set_key_to_cache(self, key, value):
             self.current_request_configuration.cache = {}
 
         self.current_request_configuration.cache[key] = value
+
+
+class CompatibleDatabaseMicrositeBackend(BaseMicrositeBackend):
```

Comment:
Why not inherit from EdunextCompatibleDatabaseMicrositeBackend ?

Reply:
Also, the name says very little. What is it compatible to? Something like TenantConfigCompatibleMicrositeBackend dice mucho más